Florian Lohoff <flo@rfc822.org> reports the following build failure:

  MODPOST 552 modules
ERROR: "free_dma" [drivers/scsi/advansys.ko] undefined!
make[5]: *** [__modpost] Error 1

--- a/drivers/scsi/Kconfig	2007-12-27 19:16:18.000000000 +0000
+++ b/drivers/scsi/Kconfig	2007-12-27 19:16:58.000000000 +0000
@@ -523,6 +523,7 @@
 	tristate "AdvanSys SCSI support"
 	depends on SCSI && VIRT_TO_BUS
 	depends on ISA || EISA || PCI
+	depends on (!MIPS && !ARM) || BROKEN
 	help
 	  This is a driver for all SCSI host adapters manufactured by
 	  AdvanSys. It is documented in the kernel source in
